src/core/game.ts

```diff
@@ -119,6 +122,16 @@ export class Game {
     });
   }
 
+  static isFinished(game: GameState) {
+    //leaving member variable isFinished since calling function in game-canvas returns true for player 1 at start
+    game.isFinished = _.size(_.filter(game.entities.players, p => p.alive)) === 1;
```

Comment:
Call this `setIsFinished` if you're setting isFinished.  a getter like `getBlah` or `isBlah` should never mutate something.